src/lib/POWERMGNT/POWERMGNT.h

```diff
@@ -20,8 +20,8 @@
 #endif
 
 #ifdef TARGET_R9M_LITE_PRO_TX
-#define MaxPower 0
-#define DefaultPowerEnum 0
+#define MaxPower 6
```

Comment:
Use those enumerations here. Other targets need to be aligned as well :) 
`#define MaxPower PWR_1000mW` and `#define DefaultPowerEnum PWR_100mW` .

Reply:
It doesn't look like the values of PowerLevels_e are consistent for different targets.

Reply:
I know, you changed all... Thanks a lot!